packages/zeal/package.json

```
@@ -0,0 +1,59 @@
+{
+  "name": "@web3-onboard/zeal",
+  "version": "2.0.0",
```

```suggestion
  "version": "2.0.0-alpha.1",
```
This will publish a package to allow us to test prior to prod release